src/tests/test_coursegrab.py

```diff
@@ -20,77 +21,83 @@ def client():
 
 @pytest.fixture
 def user():
-    _, user = users_dao.create_user("test@email.com", "First", "Last")
+    user = users_dao.create_user("test@email.com", "First", "Last")
```

Comment:
seems pretty straightforward, would probs be cleaner if we just returned `users_dao.create_user...`